units/crons.go

```diff
@@ -501,13 +501,13 @@ func PopulateAgentDeployJobs(env evergreen.Environment) amboy.QueueOperation {
 		if err != nil {
 			return errors.WithStack(err)
 		}
-
 		// don't do this more than once a minute:
-		ts := util.RoundPartOfMinute(30).Format(tsFormat)
-		catcher := grip.NewBasicCatcher()
+		ts := util.RoundPartOfHour(10).Format(tsFormat)
```

Comment:
This seems like it's too infrequent?

Reply:
It's fine because if the attempt id changes, we'll retry. the time just lets us retry every 10 minutes (e.g. static hosts) when things with the queue go pear shaped.